tests/flags/flags.go

```diff
@@ -73,6 +75,8 @@ func init() {
 	flag.StringVar(&PathToTestingInfrastrucureManifests, "path-to-testing-infra-manifests", "manifests/testing", "Set path to testing infrastructure manifests")
 	flag.StringVar(&PreviousReleaseTag, "previous-release-tag", "", "Set tag of the release to test updating from")
 	flag.StringVar(&PreviousReleaseRegistry, "previous-release-registry", "quay.io/kubevirt", "Set registry of the release to test updating from")
+	flag.StringVar(&PreviousUtilityRegistry, "previous-utility-container-registry", KubeVirtRepoPrefix, "Set registry of the utility containers to test updating from")
```

Comment:
The newly introduced flags here `PreviousUtility{Registry,Tag}` seem to duplicate the already existing ones: `PreviousRelease{Registry,Tag}`. Is there a reason not to reuse those? Do we have a usecase when both `PreviousRelease*` and `PreviousUtilities*` might be needed?

> PreviousReleaseRegistry, and have different tagging rules

What do you mean by `different tagging rules`? Maybe it would be better to adapt the tagging rules so the existing flags support both sha and tag?

Reply:
> it is quite likely that the utility containers for previous test release are in different repo 

ah, I probably overlooked that part from the description. I see the problem now.

Reply:
Exactly! The new introduced tags are for utility containers like https://github.com/kubevirt/kubevirt/blob/main/tests/operator_test.go#L962 (cirros), while `PreviousRelease{Registry,Tag}` are for kubevirt infrastructure containers like virt-controller/virt-api/etc. 
If both kinds of containers are put in the same registry, then we won't need the new tags. However, sometimes, it is possible that the two kinds of containers are put in different registries. This PR can help to facilitate this.